README.md

```diff
@@ -1,2 +1,49 @@
-# bravo-fe
```

Comment:
Add documentation around the workflow. what are all the steps involved to add a new feature?

1. create a route
2. create component
3. create action
4. create reducer 
 etc. etc.


Be precise but don't leave anything to imagination.